packages/frontend-plugin-api/src/components/ExtensionError.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+// eslint-disable-next-line @backstage/no-relative-monorepo-imports
+import { useApp } from '../../../core-plugin-api/src/app/useApp';
+
+/** @public */
+export function ExtensionError(props: { error: Error }) {
+  const { error } = props;
+  const app = useApp();
+  const { BootErrorPage } = app.getComponents();
```

Comment:
Could this cause an infinity loop too?

Reply:
I have the feeling that this component should entirely replace the legacy `BootErrorPage` component. Any thoughts?

Reply:
Component remove, see https://github.com/backstage/backstage/pull/20849#discussion_r1398824525